wagtail/admin/templates/wagtailadmin/reports/listing/_list_page_report.html

```diff
@@ -1,10 +1,10 @@
 {% load i18n wagtailadmin_tags %}
 <table class="listing {% block table_classname %}{% endblock %}">
-    <col />
+    <col width="35%" />
     <col width="12%" />
     <col width="12%" />
     <col width="12%" />
-    <col width="10%" />
+    <col />
```

Comment:
We ought to remove our usages of `width` in cols eventually, as it's deprecated: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/col#width

However, it's currently used in a lot of places, and it's also built into the tables UI framework. We need to approach it more carefully, so I opted to just make small tweaks to the `width` values in the meantime.